src/fastoad/models/performances/mission/openmdao/tests/test_with_custom_segment.py

```diff
@@ -36,7 +36,7 @@ def cleanup():
 
 @RegisterSegment("test_segment_A")
 @dataclass
-class TestSegment(AbstractFlightSegment):
+class SegmentForTest(AbstractFlightSegment):
```

Comment:
Why the name change ?

Reply:
pytest considers `Test*` classes as [items to be tested](https://docs.pytest.org/en/7.1.x/getting-started.html#group-multiple-tests-in-a-class) (and probably we should have used this feature in some of our tests)